spec/version_calculator_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+describe Fastlane::Wpmreleasetoolkit::Versioning::VersionCalculator do
+  describe 'calculates the next version number' do
+    it 'increments the major version by 1 and sets the minor, patch, and build number to 0' do
+      version = Fastlane::Models::AppVersion.new(19, 3, 1, 1)
+      bumped_version = described_class.new.next_major_version(after: version)
+      expect(bumped_version.to_s).to eq('20.0.0.0')
+    end
+
+    it 'increments the minor version by 1 and sets the patch and build number to 0' do
+      version = Fastlane::Models::AppVersion.new(19, 3, 1, 1)
+      bumped_version = described_class.new.next_minor_version(after: version)
+      expect(bumped_version.to_s).to eq('19.4.0.0')
+    end
+
+    it 'increments the patch version by 1 and sets the build number to 0' do
+      version = Fastlane::Models::AppVersion.new(19, 3, 1, 1)
+      bumped_version = described_class.new.next_patch_version(after: version)
+      expect(bumped_version.to_s).to eq('19.3.2.0')
+    end
+
+    it 'increments the build number by 1' do
+      version = Fastlane::Models::AppVersion.new(19, 3, 1, 1)
+      bumped_version = described_class.new.next_build_number(after: version)
+      expect(bumped_version.to_s).to eq('19.3.1.2')
+    end
```

Comment:
I wonder if it wouldn't make sense to use `>9` values for minor, patch and build number in those examples, instead of `3,1,1`, to ensure that the implementation of our `VersionCalculator` class doesn't assume anything about min/max values of those (like `MarketingVersionCalculator` does), and still works well with number of more than one digits.

Reply:
Addressed in https://github.com/wordpress-mobile/release-toolkit/pull/512/commits/f18258feed621c09161ee2c9dc180ba8d1517199